canvas.gemspec

```
@@ -22,6 +22,6 @@ Gem::Specification.new do |s|
   s.add_dependency "nokogiri", "~> 1.13"
   s.add_dependency "cli-ui", "~> 1.5"
   s.add_dependency "liquid", "~> 5.3"
-  s.add_dependency "dartsass-rails", "~> 0.4.0"
+  s.add_dependency "dartsass-rails", ">= 0.4.0", "< 0.6.0"
```

Should we loosen the dependency here? We don't specify a version at all in the main app.

---

That's a fair question. I could loosen the pessimistic lock to 
```suggestion
  s.add_dependency "dartsass-rails", "~> 0.4"
```
instead to maintain reasonable confidence. What do you think?